catlas/cache_utils.py

```diff
@@ -164,7 +133,7 @@ def wrapper(*args, **kwargs):
         result = NO_CACHE_RESULT
 
         cache = SqliteSingleThreadDict(
-            folder + db_loc,
+            folder + "/" + db_loc,
```

Comment:
could we make this agnostic? it would be nice if it worked if the user specifies a path with a trailing slash as well

Reply:
I believe it works regardless (`~//temp` is valid)